README.md

```diff
@@ -9,11 +9,9 @@ VMAF is a perceptual video quality assessment algorithm developed by Netflix. VM
 
 ## News
 
+- (2/27/20) We have changed VMAF's license from Apache 2.0 to [BSD+Patent](https://opensource.org/licenses/BSDplusPatent), a more permissive license compared to Apache that also includes an express patent grant. VMAF has been increasingly used in a number of open-source multimedia projects. However, it posed a problem to more deeply integrate VMAF into these projects to perform advanced tasks. Many of the open-source projects are licensed under GPLv2/LGPLv2, and the old Apache 2.0 license was incompatible with GPLv2/LGPLv2. Due to this incompatibility, when integrating VMAF source code into these projects, the license automatically got bumped to GPLv3/LGPLv3, which could prove challenging for downstream projects who might want to integrate VMAF. Changing VMAF’s license to BSD+Patent have resolved this issue, because it is compatible with GPLv2/LGPLv2.
+- (2/27/20) We made a few changes in a recent refactoring effort: 1) migrated the build system from makefile to meson, 2) restructured the code, and 3) introduced a new API with the associated library `libvmaf_rc` and executable `vmaf_rc`, co-existing with the current `libvmaf` and `vmafossexec`, all under `libvmaf/build`. The new API is designed for better interoperrability with encoding optimization. We will deprecate the old API on a future date.
```

Comment:
Same thing here. Call it a release candidate API.

Reply:
Done.